doc/source/user_guide/scale.rst

```diff
@@ -0,0 +1,343 @@
+result. Dask knows that the return type of a ``pandas.Series.value_counts``
+is a pandas Series with a certain dtype and a certain name. So the Dask version
+returns a Dask Series with the same dtype and the same name.
+
+To get the actual result you can call ``.compute()``.
+             
+.. ipython:: python
+             
+   %time ddf['name'].value_counts().compute()
+
+At that point, the full task graph (reading in data, selecting the columns,
+doing the ``value_counts``) is executed *in parallel*. You get back the same
+thing you'd get back from pandas, in this case a concrete pandas Series with the
+count of each ``name``.
+
+By default, ``dask.dataframe`` operations use a threadpool to do operations in
+parallel. We can also connect to a cluster to distribute the work on many
+machines. In this case we'll connect to a local "cluster" made up of several
+processes on this single machine.
+
+.. ipython:: python
+
+   from dask.distributed import Client, LocalCluster
+   
+   cluster = LocalCluster()
+   client = Client(cluster)
+   client
+
+Once this ``client`` is created, all of Dask's computation will take place on
+the cluster (which is just processes in this case).
+
+Dask implements the most used parts of the pandas API. For example, we can do
+a familiar groupby aggregation.
+
+.. ipython:: python
+
+   %time ddf.groupby('name')[['x', 'y']].mean().compute().head()
+
+The grouping and aggregation is done out-of-core and in parallel.
+
+When Dask knows the ``divisions`` of a dataset, certain optimizations are
+possible. When reading parquet datasets written by dask, the divisions will be
+known automatically. In this case, since we created the parquet files manually,
+we need to supply the divisions manually.
+
+.. ipython:: python
+
+   divisions = tuple(pd.to_datetime(starts)) + (pd.Timestamp(ends[-1]),)
+   ddf.divisions = divisions
+   ddf
+
+Now we can do things like fast random access with ``.loc``.
+
+.. ipython:: python
+
+   ddf.loc['2002-01-01 12:01':'2002-01-01 12:05'].compute()
+
+Dask knows to just look in the 3rd partition for selecting values in `2002`. It
+doesn't need to look at any other data.
+
+Many workflows involve a large amount of data and processing it in a way that
+reduces the size to something that fits in memory. In this case, we'll resample
+to daily frequency and take the mean. Once we've taken the mean, we know the
+results will fit in memory, so we can safely call ``compute`` without running
+out of memory. At that point it's just a regular pandas object.
+
+.. ipython:: python
+
+   @savefig dask_resample.png
+   ddf[['x', 'y']].resample("1D").mean().cumsum().compute().plot()
+
+These Dask examples have all be done using multiple processes on a single
+machine. Dask can be `deployed on a cluster
+<https://docs.dask.org/en/latest/setup.html>`_ to scale up to even larger
+datasets.
+
+You see more dask examples at https://examples.dask.org.
+
+Use chunking
```

Comment:
would move chuncking up before dask (as its a smaller section can get lost)

Reply:
*maybe* can mention some of the ecosystem libraries: vaex, modin, spark

Reply:
Originally I had the chunking earlier. I moved it down here, since I'm not sure I would ever recommend it as an option...

Perhaps I'll move it up, with the caveat that there are easier ways (Dask, etc.). And maybe show value counts, but say how difficult doing a `groupby` would be out of core.

Reply:
yeah was just a thought, though chunking is pretty appropriate for things like file-to-file transformations (e.g. csv -> parquet) for example